packages/react-components/src/organisms/EngagementTable.tsx

```diff
@@ -71,54 +84,346 @@ const iconStyles = css({
+                description="Unique Speakers Key Personnel"
+                sortingOptions={[
+                  {
+                    key: 'uniqueSpeakersKeyPersonnelDesc',
+                    label: 'Sort highest to lowest',
+                    Icon: HashtagIcon,
+                    iconTitle:
+                      'Unique Speakers Key Personnel Numerical Descending Sort Icon',
+                    onClick: onUniqueSpeakersKeyPersonnelSortClick,
+                  },
+                  {
+                    key: 'uniqueSpeakersKeyPersonnelAsc',
+                    label: 'Sort lowest to highest',
+                    Icon: HashtagIcon,
+                    iconTitle:
+                      'Unique Speakers Key Personnel Numerical Ascending Sort Icon',
+                    onClick: onUniqueSpeakersKeyPersonnelSortClick,
+                  },
+                  {
+                    key: 'uniqueSpeakersKeyPersonnelPercentageDesc',
+                    label: 'Sort highest to lowest',
+                    Icon: PercentageIcon,
+                    iconTitle:
+                      'Unique Speakers Key Personnel Percentage Numerical Descending Sort Icon',
+                    onClick: onUniqueSpeakersKeyPersonnelPercentageSortClick,
+                  },
+                  {
+                    key: 'uniqueSpeakersKeyPersonnelPercentageAsc',
+                    label: 'Sort lowest to highest',
+                    Icon: PercentageIcon,
+                    iconTitle:
+                      'Unique Speakers Key Personnel Percentage Numerical Ascending Sort Icon',
+                    onClick: onUniqueSpeakersKeyPersonnelPercentageSortClick,
+                  },
+                ]}
+              />
             </span>
-            <p css={rowValueStyles}>{row.uniqueKeyPersonnelCount} </p>
           </div>
-        ))}
-      </div>
-    </Card>
-  </>
-);
+          {data.map((row) => (
+            <div key={row.id} css={[rowStyles]}>
+              <span css={[titleStyles, rowTitleStyles]}>Team</span>
+              <p css={iconStyles}>
+                <Link href={network({}).teams({}).team({ teamId: row.id }).$}>
+                  {row.name}
+                </Link>
+
+                {row.inactiveSince && <InactiveBadgeIcon />}
+              </p>
+              <span css={[titleStyles, rowTitleStyles]}>Members</span>
+              <p css={rowValueStyles}>{row.memberCount} </p>
+              <span css={[titleStyles, rowTitleStyles]}>Events</span>
+              <p css={rowValueStyles}>{row.eventCount} </p>
+
+              <span css={[titleStyles, rowTitleStyles]}>Total Speakers</span>
+              <p css={rowValueStyles}>{row.totalSpeakerCount} </p>
+
+              <span css={[titleStyles, rowTitleStyles]}>
+                Unique Speakers: All Roles
+              </span>
+              <p css={rowValueStyles}>
+                {`${row.uniqueAllRolesCount} (${row.uniqueAllRolesCountPercentage}%)`}{' '}
```

Comment:
Why are we separating the last blank space?

Reply:
Oh, it wasn't supposed to have this blank space, I thought I added it by mistake. I've just fixed it here [review: remove extra space](https://github.com/yldio/asap-hub/pull/4339/commits/8b9782c37cfadfa812933832a73dfe5b8b3ca785). Thanks :)